requirements.txt

```diff
@@ -1,33 +1,33 @@
-astropy==5.0.4
+astropy==5.1
 astroquery==0.4.6
 bandit==1.7.4
 beautifulsoup4==4.11.1
-bokeh==2.4.2
+bokeh==2.4.3
 codecov==2.1.11
-cryptography==36.0.0
-django==3.2.5
+cryptography==38.0.1
+django==4.1
 flake8==4.0.1
 inflection==0.5.1
-ipython==8.2.0
-jinja2==3.0.3
-jsonschema==4.4.0
-jwst==1.4.6
-matplotlib==3.5.1
+ipython==8.4.0
+jinja2==3.1.2
+jsonschema==4.16.0
+jwst==1.8.2
+matplotlib==3.5.3
 nodejs==0.1.1
-numpy==1.21.5
-numpydoc==1.2
-pandas==1.4.2
+numpy==1.23.3
+numpydoc==1.4
+pandas==1.4.4
 psycopg2-binary==2.8.6
-pysiaf==0.15.0
-pytest==7.1.1
+pysiaf==0.18.0
+pytest==7.1.2
 pytest-cov==3.0.0
-pyvo==1.3
-scipy==1.7.3
-sphinx==4.4.0
+pyvo==1.4
+scipy==1.9.1
+sphinx==5.0.2
 sphinx_rtd_theme==0.4.3
-sqlalchemy==1.4.32
-stdatamodels==0.4.0
-stsci_rtd_theme==0.0.2
+sqlalchemy==1.4.39
+stdatamodels==0.4.3
```

Comment:
Is there a reason we need datamodels in the pip requirements but not in the conda env files?

Reply:
I'm not sure. I just kept the entry where it was in the file. Looks like 0.4.3 is present both in conda and pypi